pkg/mainflow.go

```diff
@@ -66,7 +66,6 @@ func Run(rawEnv map[string]string, namespace string) error {
 
 func printConfig(checkupConfig config.Config) {
 	log.Println("Using the following config:")
-	log.Printf("%q: %q", config.NUMASocketParamName, fmt.Sprintf("%d", checkupConfig.NUMASocket))
```

Comment:
Please merge this commit with the one removing the NUMA parameter from config.

Reply:
DONE